server/src/main/java/com/cloud/storage/StorageManagerImpl.java

```diff
@@ -903,6 +918,20 @@ public PrimaryDataStoreInfo createPool(CreateStoragePoolCmd cmd) throws Resource
         }
 
         Map<String, String> details = extractApiParamAsMap(cmd.getDetails());
+        if (details.containsKey("nfsopts")) {
+            Long accountId = cmd.getEntityOwnerId();
+            if (!_accountMgr.isRootAdmin(accountId)) {
```

Comment:
Do we need this specific check, createStoragePoolCmd is anyways a root admin command only ?

Reply:
yes, it is not required here. Removed